MTM-44129 Added related topics to Cockpit section (#594)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/managing-assets.md

    Use the navigator, to navigate through the asset hierarchy. In the navigator, top-level groups are shown in the **Groups** menu at top-level. Subassets are shown under its higher-level group."
---

{{< c8y-admon-related >}}

> - The [Concepts guide](/concepts/domain-model#overview) for details on the {{< product-c8y-iot >}}'s domain model.

Reviewer: ```suggestion
- The [Concepts guide](/concepts/domain-model#overview) for details on {{< product-c8y-iot >}}'s domain model.
```

- [Device Management](/users-guide/device-management#overview) in the *User guide* for details on working with devices in {{< product-c8y-iot >}}.
- The [Protocol integration guide](/protocol-integration/#overview) for more information on using protocols in {{< product-c8y-iot >}}.

> - The [Web SDK guide](/web/#overview) for more information on working with Web SDK in the {{< product-c8y-iot >}} environment.

Reviewer: ```suggestion
- The [Web SDK guide](/web/#overview) for more information on working with the Web SDK in the {{< product-c8y-iot >}} environment.
```

---

{{< c8y-admon-related >}}

> - The [Concepts guide](/concepts/domain-model#overview) for details on {{< product-c8y-iot >}}'s domain model.

Reviewer 2: ```suggestion
- [Cumulocity IoT´s domain model](/concepts/domain-model#overview) in the *Concepts guide* for details on {{< product-c8y-iot >}}'s domain model.
```

> - [Device Management](/users-guide/device-management#overview) in the *User guide* for details on working with devices in {{< product-c8y-iot >}}.

Reviewer 2: ```suggestion
- [Device Management](/users-guide/device-management#overview) for details on working with devices in {{< product-c8y-iot >}}.
```

> - The [Protocol integration guide](/protocol-integration/#overview) for more information on using protocols in {{< product-c8y-iot >}}.

Reviewer 2: Why would you link to the Protocol integration guide here?

Author: ```suggestion
```

Author: @BeateRixen I have removed it.

> - The [Web SDK guide](/web/#overview) for more information on working with the Web SDK in the {{< product-c8y-iot >}} environment.

Reviewer 2: And think I commented this earlier, why do we have a generic link to the Web SDK guide here?

Author: ```suggestion
- [Tutorials > Add a Jest-based unit test](/guides/web/tutorials/#add-a-unit-test) in the *Web SDK guide* for more information on working with custom tabs in the {{< product-c8y-iot >}} environment.
```

Author: @BeateRixen I have modified the link to a specific endpoint.
